google/cloud/pubsublite/endpoint.h

```diff
@@ -0,0 +1,35 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef GOOGLE_CLOUD_CPP_GOOGLE_CLOUD_PUBSUBLITE_ENDPOINT_H
+#define GOOGLE_CLOUD_CPP_GOOGLE_CLOUD_PUBSUBLITE_ENDPOINT_H
+
+#include "google/cloud/options.h"
+#include "google/cloud/status_or.h"
+#include "google/cloud/version.h"
+#include <string>
+
+namespace google {
+namespace cloud {
+namespace pubsublite {
+GOOGLE_CLOUD_CPP_INLINE_NAMESPACE_BEGIN
+
+StatusOr<std::string> EndpointFromZone(std::string const& zone_id);
```

Comment:
I think we also want one for regions. So like:

```cc
/// A region looks like: `us-east4`
StatusOr<std::string> EndpointFromRegion(std::string const& region_id);
/// A zone looks like: `us-east4-c`
StatusOr<std::string> EndpointFromZone(std::string const& zone_id);
```

(here's why I think that). Create Topic UI:
![image](https://user-images.githubusercontent.com/23088558/152652480-7e873a00-0823-4f8c-9e40-40d021476470.png)

I made one of each, and they are displayed like this:
![image](https://user-images.githubusercontent.com/23088558/152652499-83790648-f64a-492f-b178-372cf036d0ea.png)

Reply:
Or maybe `EndpointFromLocation`? that could get messy though.

Reply:
Thanks.  I added the new helper, but left the quickstart as-is.